utility/src/main/java/org/oppia/android/util/parser/html/ConceptCardTagHandler.kt

```diff
@@ -52,10 +53,18 @@ class ConceptCardTagHandler(
   }
 
   override fun getContentDescription(attributes: Attributes): String? {
-    val skillId = attributes.getJsonStringValue(CUSTOM_CONCEPT_CARD_SKILL_ID)
     val text = attributes.getJsonStringValue(CUSTOM_CONCEPT_CARD_TEXT_VALUE)
-    return if (!skillId.isNullOrBlank() && !text.isNullOrBlank()) {
-      "$text concept card $skillId"
+    return if (!text.isNullOrBlank()) {
+      text
     } else ""
   }
+
+  /** Application-injectable factory for creating [ConceptCardTagHandler]s). */
+  class Factory @Inject constructor() {
+    fun createConceptCardLinkClickListener(): ConceptCardLinkClickListener {
```

Comment:
I believe this should have a KDoc since it is a public function. KDoc check seems to not have caught this.